src/providers/routes.provider.tsx

```diff
@@ -84,6 +85,7 @@ const paths: PathObject = {
     myApp: "/my-application",
     ticket: "/ticket/:ticketId",
     perks: "/perks",
+    location: "/location"
```

Comment:
This new path can be removed since `/location` in the navbar should be the global laz location right?